ui/dashboards/src/context/DashboardProvider/DashboardAppSlice.tsx

```diff
@@ -31,7 +32,7 @@ export interface DashboardAppSlice {
   closePanelGroupDialog: () => void;
 }
 
-export const createDashboardAppSlice = immer<DashboardAppSlice>((set) => ({
+export const createDashboardAppSlice: StateCreator<DashboardAppSlice, Middleware, [], DashboardAppSlice> = (set) => ({
```

Comment:
Yeah, it's basically the function signature for a "slice" of a store. It basically makes it so that the `get` and `set` functions that are passed as arguments get the right types.

Reply:
ohh cool i thought we had to also call `immer` middleware here to use the immer syntax for modifying object